cli/cmd/create_project.go

```diff
@@ -112,14 +127,38 @@ keptn create project PROJECTNAME --git-user=GIT_USER --git-remote-url=GIT_REMOTE
 			project.GitToken = *createProjectParams.GitToken
 			project.GitRemoteURL = *createProjectParams.RemoteURL
 
+			if *createProjectParams.GitProxyURL != "" && strings.HasPrefix(*createProjectParams.RemoteURL, "ssh://") {
+				return errors.New(gitErrMsg)
+			}
+
+			if *createProjectParams.GitProxyURL != "" && *createProjectParams.GitProxyScheme == "" {
+				return errors.New(gitErrMsg)
```

Comment:
here we also need a specific error message, since `gitErrMsg` only explains that git-token and private key cannot be used at the same time